src/main/java/ch/njol/skript/SkriptConfig.java

```diff
@@ -105,7 +105,7 @@ public class SkriptConfig {
 						break;
 					case "stable":
 						// TODO a better option would be to check that it is not a pre-release through GH API
-						channel = new ReleaseChannel((name) -> !name.contains("pre"), t);
+						channel = new ReleaseChannel((name) -> !(name.contains("pre") || name.contains("beta")), t);
```

Comment:
Do you think it would be a good idea to assume that stable releases do not contain a hyphen? This would ensure more flexibility for the future. The build.gradle could also be updated to assume prerelease when a hyphen is in the version tag.

Reply:
I think we are better off using HTML comments to provide more precise data. github markdown allows it

Reply:
unfortunately, the update checker system currently only passes the version of the release, rather than the entire release manifest. i do have plans to update this system though :)

Reply:
> Do you think it would be a good idea to assume that stable releases do not contain a hyphen? This would ensure more flexibility for the future. The build.gradle could also be updated to assume prerelease when a hyphen is in the version tag.

yeah i think this is a good idea